poems-subject.html

```diff
@@ -0,0 +1,168 @@
+        <!--[if lt IE 8]>
+            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+		
+    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+      <div class="container">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="index.html">MegaPoem</a>
+        </div>
+        <div id="navbar" class="navbar-collapse collapse">
+		  <ul class="nav navbar-nav">
+			<li class="dropdown active">
+			  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Poems <span class="caret"></span></a>
+			  <ul class="dropdown-menu">
+				 
+				 
+				<li><a href="poems.html">Title</a></li>
+				<li><a href="poems-subject.html">Subject Matter</a></li>
+				<li><a href="#">Popularity</a></li>
+				
+				<li><a href="#">Location</a></li>
+			  </ul>
+			</li>
+			<li><a href="#">Magazine</a></li>
+			<li><a href="#">About</a></li>  
+			<!-- <li class="dropdown">
+			  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Poems <span class="caret"></span></a>
+			  <ul class="dropdown-menu">
+				<li><a href="#">Subject Matter</a></li>
+				<li><a href="#">Aurther</a></li>
+				<li><a href="#">Something else here</a></li>
+				<li role="separator" class="divider"></li>
+				<li><a href="#">Separated link</a></li>
+				<li role="separator" class="divider"></li>
+				<li><a href="#">One more separated link</a></li>
+			  </ul>
+			</li> -->
+		  </ul>	
+          <ul class="nav navbar-nav navbar-right">
+        		<li><a href="#"><span class="glyphicon glyphicon-user" aria-hidden="true"></span> Sign In</a></li>
+      	  </ul>  
+        </div><!--/.navbar-collapse -->
+      </div>
+    </nav>
+
+    <!-- Main jumbotron for a primary marketing message or call to action -->
+    <div class="jumbotron">
+      <div class="container">
+        <h1>Katie</h1>— by HENRY TIMROD<p></p>
+        <p>It may be through some foreign grace, 
+And unfamiliar charm of face; 
+It may be that across the foam 
+Which bore her from her childhood’s home, 
+By some strange spell, my Katie brought, 
+Along with English creeds and thought...</p>
+        
+      </div>
+    </div>
+
+    <div class="container container-home">
+      <!-- Example row of columns -->
+      <div class="row">
+        <div class="col-md-4">
+          <ul class="list-unstyled">
+			  <li><span class="h2">Poems </span>by<li>
```

Comment:
Both are good. I was leaning toward 'theme' because it's a tad broader. We might have a category in the future that doesn't quite fit as a 'subject' but it would probably fit as a 'theme.'